cmd/proxima/proxima.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"flag"
+	"github.com/Symantec/Dominator/lib/fsutil"
+	"github.com/Symantec/scotty/lib/apiutil"
+	"github.com/influxdata/influxdb/models"
+	"github.com/influxdata/influxdb/uuid"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+)
+
+func setHeader(w http.ResponseWriter, r *http.Request, key, value string) {
+	r.Header.Set(key, value)
+	w.Header().Set(key, value)
+}
+
+func uuidHandler(inner http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		uid := uuid.TimeUUID()
+		setHeader(w, r, "Request-Id", uid.String())
+		setHeader(w, r, "X-Influxdb-Version", "0.13.0")
+		inner.ServeHTTP(w, r)
+	})
+}
+
+type seriesListType struct {
+	Series []models.Row `json:"series"`
+}
+
+type resultListType struct {
+	Results []seriesListType `json:"results"`
+}
+
+func main() {
+	fConfigFile := flag.String("config", "./proxima.config", "config file")
+	fPort := flag.String("port", ":8086", "listen port")
+	flag.Parse()
+	executer := newExecuter()
+	logger := log.New(os.Stderr, "", log.LstdFlags)
+	changeCh := fsutil.WatchFile(*fConfigFile, logger)
```

Comment:
Can you follow the pattern that the various Dominator servers (and I think Scotty as well) use? Flags defined in a var table, using the logbuf package, creating a front page dashboard, adding metrics with tricorder?